cds_migrator_kit/videos/scripts/copy_files.py

```diff
@@ -13,7 +13,10 @@
 import os
 import re
 import shutil
+import subprocess
 import sys
+from collections import defaultdict
+from multiprocessing.dummy import Pool as ThreadPool  # threads for I/O bound tasks
```

Comment:
this one is not used right?

Reply:
If you can enable threading by config, so we can test with one vs multiple yes :)